samtools.1

```diff
@@ -1,11 +1,12 @@
-.TH samtools 1 "2 February 2015" "samtools-1.2" "Bioinformatics tools"
+.TH samtools 1 "20 April 2015" "samtools-1.3" "Bioinformatics tools"
```

Comment:
I bump these when I make the release… so will probably temporarily undo this line when this is merged.